lexicons/app/bsky/admin/takeModerationAction.json

```diff
@@ -0,0 +1,30 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.admin.takeModerationAction",
+  "defs": {
+    "main": {
+      "type": "procedure",
+      "description": "Take a moderation action on an actor.",
+      "input": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["action", "subject", "reason", "createdBy"],
+          "properties": {
+            "action": {"type": "string", "knownValues": ["app.bsky.admin.actionTakedown"]},
+            "subject": {"type": "union", "refs": ["app.bsky.actor.ref"]},
```

Comment:
I don't think the declaration CID is actually relevant to the takedown. 

Like if I've banned a certain user, & they change their repo declaration to a "scene repo", I probably still want it banned